spec/rails_helper.rb

```diff
@@ -1,25 +1,18 @@
 # frozen_string_literal: true
 
-require 'spec_helper'
 require File.expand_path('../config/environment', __dir__)
 abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'rspec/rails'
 require 'pundit/rspec'
 require 'pundit/matchers'
 require 'database_cleaner'
-# frozen_string_literal: true
-
 require 'spec_helper'
-require File.expand_path('../config/environment', __dir__)
-abort("The Rails environment is running in production mode!") if Rails.env.production?
-require 'rspec/rails'
-require 'pundit/rspec'
```

Comment:
Why did you remove all those requires?

Reply:
Reason being, in the original file they were duplicated, as can be seen here.